### PR TITLE
doc-builder -> doc-build

### PR DESCRIPTION
**Pull request by LysandreJik**
Changes the repository on which the documentation builds will be pushed.

### CODE REVIEW

**Comment by mishig25** [comment]
Just want to make sure [this feature](https://huggingface.slack.com/archives/C02GLJ5S0E9/p1639171159299300) is unrelated to this PR

File: .github/workflows/build_documentation.yml [hunk 1]
```diff
@@ -77,13 +77,14 @@ jobs:
 
       - name: Make documentation
         run: |
-          doc-builder build transformers transformers/docs/source --build_dir doc-builder/build --notebook_dir notebooks/transformers_doc --clean
+          mkdir doc-build/temp
```
Comment (julien-c) [inline .github/workflows/build_documentation.yml:80]:
```suggestion
```
do we still need this?

**Comment by LysandreJik** [comment]
> Just want to make sure this feature is unrelated to this PR

I have not touched the notebook part of the PR, the notebooks will still get automatically updated AFAICT

File: .github/workflows/build_documentation.yml (hunk 1 above)
Comment (LysandreJik) [reply .github/workflows/build_documentation.yml]: No, we don't! Thank you

**Pull request merged**